soap/src/java/com/zimbra/soap/type/SearchSortBy.java

```diff
@@ -21,14 +21,52 @@
 
 import javax.xml.bind.annotation.XmlEnum;
 
+import com.zimbra.common.mailbox.ZimbraSortBy;
 import com.zimbra.common.service.ServiceException;
 import com.zimbra.common.zclient.ZClientException;
 
 @XmlEnum
 public enum SearchSortBy {
     // case must match protocol
-    dateDesc, dateAsc, subjDesc, subjAsc, nameDesc, nameAsc, durDesc, durAsc, none,
-    taskDueAsc, taskDueDesc, taskStatusAsc, taskStatusDesc, taskPercCompletedAsc, taskPercCompletedDesc, rcptAsc, rcptDesc;
+    dateDesc(ZimbraSortBy.dateDesc),
```

Comment:
The WSDL is the same after this change:
<xs:simpleType name="searchSortBy">
<xs:restriction base="xs:string">
<xs:enumeration value="dateDesc"/>
<xs:enumeration value="dateAsc"/>
<xs:enumeration value="subjDesc"/>
<xs:enumeration value="subjAsc"/>
<xs:enumeration value="nameDesc"/>
<xs:enumeration value="nameAsc"/>
<xs:enumeration value="durDesc"/>
<xs:enumeration value="durAsc"/>
<xs:enumeration value="none"/>
<xs:enumeration value="taskDueAsc"/>
<xs:enumeration value="taskDueDesc"/>
<xs:enumeration value="taskStatusAsc"/>
<xs:enumeration value="taskStatusDesc"/>
<xs:enumeration value="taskPercCompletedAsc"/>
<xs:enumeration value="taskPercCompletedDesc"/>
<xs:enumeration value="rcptAsc"/>
<xs:enumeration value="rcptDesc"/>
</xs:restriction>